### DISCUSSION TITLE
Open Stream Using SDP File

### DISCUSSION THREAD

**Discussion by Ricardo-Covarrubias** [SuRGeoNix/Flyleaf#529] [Q&A]
This project has been very helpful!
Currently I'm trying to capture an RTP Stream, and I'm able to use the following command from ffplay to pass the SDP configuration file like so:
`ffplay "./config.sdp" -protocol_whitelist "file,udp,rtp" -fflags nobuffer -flags low_delay`
I've tried to look at the project and other questions and issues but can't quite find how to pass this file, or the SDP configurations into the FlyLeaf component, I've tried giving the Player.Open/OpenAsync the file location but it throws an error:
`[avformat_open_input] Invalid data found when processing input (-1094995529)`

Here's the actual config.sdp file I'm using:
```
v=0
o=- 0 0 IN IP4 127.0.0.1
s=Media Server
c=IN IP4 127.0.0.1
t=0 0
a=tool:libavformat 61.9.100
m=video 1234 RTP/AVP 96
a=rtpmap:96 H265/90000
a=fmtp:96 sprop-vps=QAEMAf//AWAAAAMAsAAAAwAAAwBarAk=; sprop-sps=QgEBAWAAAAMAsAAAAwAAAwBaoAWCAeFja5JMvkA=; sprop-pps=RAHA8vA8kA==
```

How am I meant to feed the SDP file / SDP Configurations into FlyLeaf? Help is greatly appreciated.

**Comment by SuRGeoNix** [comment]
Hi @Ricardo-Covarrubias

Sorry for being late... SDP should be supported directly from FFmpeg so no special configuration is required.
Maybe the issue you had was with the libavformat 61.9.100
If you still have issues, let me know and I will look into it further.
